### ISSUE TITLE
Expanding a proposal throws a "not a function" error

### ISSUE THREAD

**Issue by mwapst** [orppst/pst-gui#229]
Strangely this seems to affect only the standard "Observing the stars" proposal, it doesn't happen if you create a new one.

The caught error says

observation.target?.map is not a function. (In 'observation.target?.map((obsTarget) => { let targetObj = proposalsData?.targets?.find((target) => target._id === obsTarget); targetObjs.push(targetObj); })', 'observation.target?.map' is undefined) 

which is declared in proposal/Overview.tsx and targets/targetPanel.tsx

**Comment by mwapst** [comment]
following Allan's advice I inserted the line "// @ts-ignore" before the offending item in overview.tsx as a temp fix which worked

**Comment by DJWalker42** [comment]
I cannot replicate this issue. 

There are potential changes to the underlying data model (allowing Observations to have multiple Targets) that have yet to be reflected in the API, [pst-api-service/issues/39](https://github.com/orppst/pst-api-service/issues/39). And the code in Overview.tsx will also need changing once this is done. I'll create a new issue, and close this issue.
